include/CoreGraphics/CGDataConsumer.h

```diff
@@ -19,7 +19,7 @@
 #import <CoreFoundation/CFURL.h>
 #import <CoreFoundation/CFData.h>
 
-typedef struct __CGDataConsumer* CGDataConsumerRef;
+typedef struct CGDataConsumer* CGDataConsumerRef;
```

Comment:
I've seen other cases where people have used CG...* vs CG...Ref so I thought we should keep it equivalent to the reference platform, which interestingly breaks pattern for this https://developer.apple.com/reference/coregraphics/cgdataconsumerref?language=objc